tests/common/experimental/test_reducers_and_aggregators.py

```diff
@@ -606,3 +609,105 @@ def test_hawq_aggregator(self, inputs, reference_output):
 
         ret_val = aggregator.aggregate()
         assert fns.allclose(ret_val, reference_output)
+
+    @pytest.mark.parametrize(
+        "ref_hist,ref_min,ref_max,ref_aggr_min,ref_aggr_max",
+        [
+            (
+                np.array(
```

Comment:
Is it really need to define np.array as parameter, can it just list?

Reply:
Fixed